dockerfile

```diff
@@ -1,8 +1,21 @@
-FROM golang:1.21.1-bullseye
+FROM golang:1.21.1 as builder
 
 WORKDIR /go/src/
+
 COPY . .
-RUN go build ./cmd/server/
-EXPOSE 8081
 
+RUN CGO_ENABLED=1 GOOS=linux go build -o ./server --ldflags '-w -s -extldflags "-static"' ./cmd/server/
+
+FROM alpine:3.17 as alpine
```

Comment:
we're using alpine 3.18 in other images, let's keep this version unified.

Reply:
Fixed